src/index.tsx

```diff
@@ -7,7 +7,7 @@ import App from "./App";
 
 const config: Config = {
   readOnlyChainId: Goerli.chainId,
-  networks: [Goerli]
+  networks: [Goerli],
```

Comment:
Mumbai?  What is this network intended to be used for?

Reply:
On Mumbai, fixed